whylog/config/abstract_config.py

```diff
@@ -75,9 +74,12 @@ def _save_filename_matcher_definition(self, matcher_definition):
 
     def get_log_type(self, front_input):
         line_source = front_input.line_source
-        for log_type in six.itervalues(self._log_types):
-            if log_type.is_belong_to_log_type(line_source):
-                return log_type
+        return next(
+            (
+                log_type for log_type in six.itervalues(self._log_types)
+                if log_type.does_belong_to_log_type(line_source)
+            ), None
+        )
```

Comment:
Sorry for confusing, but I was'nt sure if rewriting this into list comprehension form would be more readable - I have proposed it only, now I'm afraid that the previous form was more readable. Please decide by your own which form is better and apply it.